packages/sanes-chrome-extension/src/utils/test/persona.ts

```diff
@@ -0,0 +1,52 @@
+import { Store } from 'redux';
```

Comment:
As far as I remember this is also duplicated, right? I prefer to use what is in `extension/model/persona/test`.